pkgs/applications/virtualization/buildkit-nix/default.nix

```
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "buildkit-nix";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "AkihiroSuda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Bdw7kYAZmRf1VOJ8y7JISQbAo0vLjWlo0j5x+VD9lSU=";
+  };
+
+  vendorSha256 = "sha256-c+VHt2uTaEQIXsmJ9TA7X5lfMxGL9yKbbnnXn4drCLU=";
+
+  meta = with lib; {
```

[Same as the upstream build.](https://github.com/AkihiroSuda/buildkit-nix/blob/8f34874e94858da7e6673332caafd950bb0e4827/bootstrap.Dockerfile#L8-L9)

```suggestion
  vendorSha256 = "sha256-c+VHt2uTaEQIXsmJ9TA7X5lfMxGL9yKbbnnXn4drCLU=";
  
  CGO_ENABLED = 0;
  
  ldflags = [ "-s" "-w" ];

  meta = with lib; {
```